ui/stories/masked-input.stories.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/extensions */
+import hbs from 'htmlbars-inline-precompile';
+import { storiesOf } from '@storybook/ember';
+import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+import notes from './masked-input.md';
+
+storiesOf('MaskedInput/', module)
+  .addParameters({ options: { showPanel: true } })
+  .addDecorator(withKnobs())
+  .add(
+    `MaskedInput`,
+    () => ({
+      template: hbs`
+        <h5 class="title is-5">Masked Input</h5>
+        <MaskedInput
```

Comment:
Is our general guideline that if we use knobs, we won't do multiple stories?

Reply:
I think knobs are preferable to multiple stories because they're interactive, yes. They're definitely preferable in the case where changing properties results in little visual difference and properties are well-named enough that the user can make a clear guess what altering then would do in the UI.

With this component the only additional story I considered was showing a `MaskedInput` with `displayOnly` set to `true` beneath one with `displayOnly` set to false, but I think using the knob to showcase that difference is simple enough.